admin/metabox/class-metabox-collapsibles-section.php

```diff
@@ -0,0 +1,67 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Generates and displays the React root element for a metabox section.
```

Comment:
This comment seems strange to me. Is it relevant to this class that we use the rendered output for a React root?